go.mod

```diff
@@ -210,6 +211,11 @@ require (
 	modernc.org/sqlite v1.23.1
 )
 
+require (
+	github.com/Masterminds/semver/v3 v3.2.0 // indirect
+	github.com/spf13/cast v1.5.1 // indirect
+)
+
 require (
```

Comment:
Please merge the two sections.

Reply:
I was wondering why `go mod tidy` added that extra section... Sure, I'll just merge them.